### PR TITLE
.NET 6 preview 3 API updates

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

**Review by safern — APPROVED** [review]
Just making sure the moniker is going to show up as Preview3 right?

**Comment by gewarren** [comment]
> Just making sure the moniker is going to show up as Preview3 right?

Good call, I've updated the display name to .NET 6 preview 3.

File: xml/System.Linq/Queryable.xml [hunk 1]
```diff
@@ -556,6 +566,12 @@
         <AssemblyVersion>2.0.0.0</AssemblyVersion>
         <AssemblyVersion>2.1.0.0</AssemblyVersion>
       </AssemblyInfo>
+      <Attributes>
+        <Attribute FrameworkAlternate="net-6.0">
+          <AttributeName Language="C#">[System.Diagnostics.CodeAnalysis.RequiresUnreferencedCode("Enumerating in-memory collections as IQueryable can require unreferenced code because expressions referencing IQueryable extension methods can get rebound to IEnumerable extension methods. The IEnumerable extension methods could be trimmed causing the application to fail at runtime.")]</AttributeName>
```
Comment (gewarren) [inline xml/System.Linq/Queryable.xml:571]: New attribute.

File: xml/System.Linq/Queryable.xml [hunk 2]
```diff
@@ -2009,6 +2053,43 @@
         <exception cref="T:System.InvalidCastException">An element in the sequence cannot be cast to type <paramref name="TResult" />.</exception>
       </Docs>
     </Member>
+    <Member MemberName="Chunk&lt;TSource&gt;">
```
Comment (gewarren) [inline xml/System.Linq/Queryable.xml:2056]: New method.

File: xml/System.Linq/Queryable.xml [hunk 3]
```diff
@@ -2613,6 +2703,43 @@
           <paramref name="source" /> or <paramref name="comparer" /> is <see langword="null" />.</exception>
       </Docs>
     </Member>
+    <Member MemberName="ElementAt&lt;TSource&gt;">
```
Comment (gewarren) [inline xml/System.Linq/Queryable.xml:2706]: New overload.

**Pull request merged**